File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/configuration/conventions/WhatAreConventions.java
package net.ravendb.clientapi.configuration.conventions;

import net.ravendb.client.document.DocumentConvention;
import net.ravendb.client.document.DocumentStore;


public class WhatAreConventions {
  public WhatAreConventions() {
    DocumentStore store = new DocumentStore();
    //region conventions_1
    DocumentConvention conventions = store.getConventions();
    //endregion
  }
}

File: Documentation/3.5/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Conventions/WhatAreConventions.cs
namespace Raven.Documentation.Samples.ClientApi.Configuration.Conventions
{
	using Client.Document;

	public class WhatAreConventions
	{
		public WhatAreConventions()
		{
			var store = new DocumentStore();

			#region conventions_1
			DocumentConvention conventions = store.Conventions;
			#endregion
		}
	}
}